app/utils/resourceUtils.js

```diff
@@ -25,10 +25,8 @@ function hasMaxReservations(resource) {
 function isOpenNow(resource) {
   const { closes, opens } = getOpeningHours(resource);
   const now = moment();
-  if (now >= moment(opens) && now <= moment(closes)) {
-    return true;
-  }
-  return false;
+
+  return now >= moment(opens) && now <= moment(closes);
```

Comment:
How about using `moment`'s own helpers `isSameOrBefore(x)` and `isSameOrAfter(x)`?

Reply:
Probably yeah. This is basically copy/pasted from the old utility functions, it's just modified to work without redux/data straight from the API.